Tryouts/Prototypes/Shell/MainWindow.xaml.cs

```diff
@@ -40,13 +41,59 @@ public partial class MainWindow : Window
         private ManifestModel config;
         private ModuleModel[]? modules;
 
+        private string commandLineURL;
+        private string[] hereIsMyCode = (Application.Current as App).CustomerCode;
+
         public MainWindow()
         {
             InitializeComponent();
 
-            config = new ManifestParser().Manifest;
-            modules = config.Modules;
-            DataContext = modules;
+            this.Loaded += MainWindow_Loaded;
+        }
+
+        private void MainWindow_Loaded(object sender, RoutedEventArgs e)
+        {
+            if (this.hereIsMyCode.Length != 0)
```

Comment:
What's the expected behavior here? If I provide the url, an empty `MainWindow` will pop up. Consider hiding the main window in this case.